PKGBUILD.MSYS2

```diff
@@ -0,0 +1,68 @@
+# Themis build script for pacman in MSYS2 environment
+#
+# Maintainer: Cossack Labs Limited <dev@cossacklabs.com>
+
+pkgname=('themis' 'themis-devel')
+pkgbase=themis
+pkgver=0.11.2
+pkgrel=1
+
+pkgdesc="Data security library for network communication and data storage"
+url="https://www.cossacklabs.com/themis/"
+license=('Apache')
+changelog=CHANGELOG.md
+
+arch=('x86_64' 'i686')
+depends=('libopenssl>=1.1.1')
+makedepends=('tar' 'gcc' 'make' 'openssl-devel>=1.1.1')
+
+# TODO: replace with an proper release tarball
+#
+# Currently a fork with MSYS2 support is used. Replace this with 'cossacklabs'
+# repository once official release tarball with MSYS2 support is available.
+source=("https://github.com/ilammy/themis/archive/$pkgver.tar.gz")
```

Comment:
ilammy/themis -> cossacklabs/themis ?)

Reply:
Mmm... I guess yeah, I'll change it right away.

My plan was to keep it that way so that running makepkg produces something, but I guess it would be better to keep the file in Git correct. Developers can tweak it locally for experiments.